helm-charts/nginx-ingress/templates/post-upgrade-job.yaml

```diff
@@ -41,8 +45,12 @@ spec:
         runAsUser: 1000
       serviceAccount: nginx-ingress-serviceaccount
       containers:
-      - name: kubectl
+      - name: post-install-job
```

Comment:
suggest to keep the `kubectl` name as hook will be triggered in both post-install and post-upgrade case.